modules/search/class.jetpack-search.php

```diff
@@ -283,6 +291,36 @@ public function load_assets() {
 		}
 	}
 
+	/**
+	 * Registers a widget sidebar for Instant Search.
+	 */
+	public function register_jetpack_instant_sidebar() {
+		$args = array(
+			'name'          => 'Jetpack Instant Search Sidebar',
+			'id'            => 'jetpack-instant-search-sidebar',
+			'description'   => 'Customize the Jetpack Instant Search overlay sidebar',
+			'class'         => '',
+			'before_widget' => '<div id="%1$s" class="widget %2$s">',
+			'after_widget'  => '</div>',
+			'before_title'  => '<h2 class="widgettitle">',
+			'after_title'   => '</h2>',
+		);
+		register_sidebar( $args );
+	}
+
+	/**
+	 * Prints Instant Search sidebar.
+	 */
+	public function print_instant_search_sidebar() {
```

Comment:
I wonder if we might need a few checks in here before outputting this into the footer. I am not 100% sure but `wp_footer` might get called on `amp` pages for example, we might want to try and think of conditions in which outputting this is not a good idea.

Reply:
I don't have much exposure to AMP, but it [seems to support `display: none;`](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/style_pages/?format=websites) without any issues. Also, I double-checked existing `wp_footer` hooks within Jetpack and didn't find anything that might conflict. 

I think it's probably safe to push this out.

@jeherve, do you have any guidance on what to look out for when printing a hidden div into the footer? This function prints a hidden widget sidebar area which is then reattached and made visible in a different section of the DOM via JavaScript.

Reply:
I'll go ahead and ship this PR for now, but I'll keep an eye on this thread.